integration/build.gradle

```diff
@@ -15,7 +15,7 @@ dependencies {
         guava = "29.0-jre"
         jackson = "2.10.2"
         jersey = "2.32"
-        kubernetesClient = "10.0.0"
+        kubernetesClient = "16.0.0"
```

Comment:
I noticed that we had a lot of compile-time dependencies on old versions of libraries (with newer versions being used as a runtime dependency). When I looked at the dependency tree, I noticed this was coming from `kubernetesClient`, which was at a version from 2020. The newer versions include a lot of dependabot upgrades (https://github.com/kubernetes-client/java/releases?page=1), and given that this is a test dependency only, upgrading seems low risk if all the tests pass.